### DISCUSSION TITLE
Will Moonlight continue to work with Apollo?

### DISCUSSION THREAD

**Discussion by RERLTuna** [ClassicOldSong/Apollo#109] [Q&A] [answered]
First of all, this is a phenomenal program, but I know in the disclaimer it is said the OG moonlight will not work with Apollo eventually, but I see Artemis is only for Android atm it seems. I use a lower end PC to stream from my higher end PC, will there eventually be an Artemis client for Windows as well? Or will moonlight continue to work with Apollo? Thank you again.

**Comment by xenophobentx** [comment]
For me , the moonlight windows client works perfectly, registering as a legacy client in apollo.

**Comment by ClassicOldSong** [comment, marked as answer]
There will be Artemis for other platforms including Windows. I'm a bit busy lately, but there will be a desktop version in the future.

**Reply by RERLTuna** [reply]
Thank you for the quick response. Awesome, can't wait to see what you have in store.

**Reply by Nemosapien** [reply]
By chance is there an update on when the Windows platform version might be coming?  Thanks for all you do.
